control-plane/roles/logging/tasks/main.yaml

```diff
@@ -12,19 +12,33 @@
         labels:
           name: "{{ logging_namespace }}"
 
-- name: Deploy loki-stack
+- name: Deploy basic auth secret for loki
+  k8s:
+    definition:
+      apiVersion: v1
+      kind: Secret
+      metadata:
+        name: loki-basic-auth
+      type: Opaque
+      data:
+        auth: "{{ logging_ingress_loki_basic_auth | b64encode }}"
+    namespace: "{{ logging_namespace }}"
+    apply: true
+  when: logging_ingress_loki_basic_auth
+
+- name: Deploy loki
   include_role:
     name: ansible-common/roles/helm-chart
   vars:
     helm_repo: "{{ logging_chart_repo }}"
-    helm_chart: loki-stack
+    helm_chart: loki
```

Comment:
As we are changing to another chart, we need to adapt the release vector version: https://github.com/metal-stack/releases/blob/develop/release.yaml#L185-L187

Reply:
Let's also add an `ACTIONS_REQUIRED` section to the PR to indicate that the old helm installation has to be uninstalled manually before deployment.